packages/opentelemetry-plugin-grpc/test/grpc.test.ts

```diff
@@ -436,8 +436,7 @@ describe('GrpcPlugin', () => {
       return provider.getTracer('default').withSpan(span, async () => {
         const rootSpan = provider.getTracer('default').getCurrentSpan();
         if (!rootSpan) {
-          assert.ok(false);
-          return; // return so typechecking passes for rootSpan.end()
+          return assert.ok(false);
```

Comment:
You don't need the return at all. `assert.ok(false)` throws an exception

Reply:
I have to otherwise it was failing - see the previous build - I have fix like this